### ISSUE TITLE
Installing pygenefinder issue

### ISSUE THREAD

**Issue by oktaykaplan** [dmnfarrell/pathogenie#3]
I am having issues installing pygenefinder on my Macintosh machine (python 3.7.0), would you have any suggestion to resolve the issue? 

pip install pygenefinder resulted in the following errors: 

ERROR: Could not find a version that satisfies the requirement pygenefinder (from versions: none)
ERROR: No matching distribution found for pygenefinder

Kind regards

**Comment by dmnfarrell** [comment]
Yes sorry I haven't yet set it up on pypi yet. You can install from the repo using:
`pip install -e git+https://github.com/dmnfarrell/pygenefinder.git#egg=pygenefinder`. I'll update the readme. However I have not tested this on a mac so you'll need to make sure you have ncbi blast, prodigal, hmmer3 and aragorn installed. I think you can install them with bioconda if you're familiar with that.

**Comment by koalap-2018** [comment]
Hi Damien,

I just ran into your package and it looks great. Thanks for sharing with the community. I am having troubles installing it tho. I have tried both your pip install the link in your comment above, and then the pip install -e git+https://github.com/dmnfarrell/pathogenie.git#egg=pathogenie. Both of which install it find in anaconda prompt but when i try to load in jupyter notebook give following errors:

ModuleNotFoundError: No module named 'pygenefinder'

or

NameError: name 'urllib' is not defined

I have python 3.6 . if you think of something that might be an easy fix i would appreciate it

thanks!

**Comment by koalap-2018** [comment]
Hi Damien, 

I believe there is a small bug in your code. the tools.py doesn have the import urllib at the begging. That seemed to fix my issue but then i ran into a new one. HTTPError: HTTP Error 404: Not Found

Let me know if you have any suggestions.

thanks!

**Comment by dmnfarrell** [comment]
Paola
That bug should be fixed. If you're running on windows you will also need the Visual Studio 2015 C++ redistributable runtime package: https://www.microsoft.com/en-us/download/details.aspx?id=48145
I've updated the README to make it clearer. Let me know if it works.

**Comment by koalap-2018** [comment]
Thanks so much for the prompt reply. I will take a look at this for sure :)

**Comment by koalap-2018** [comment]
Hi Damien,

I believe your suggestion did the trick. Thanks! I havent ran much on it but will get to it soon.  I only ran this:  featdf,recs = pathogenie.app.run_annotation('gisaid_cov2020_sequences.fasta', kingdom='viruses') and seemed to give expeccted output.

Just a couple naive questions... 
1.pathogenie and pygenefinder are exact same right?
2.once i get the sequenes from GISAID you basically have just on file with all .FASTA so you can read and load from this function?: 
gisrecs = load_deduplicated_sequences('gisaid_cov2020_sequences.fasta')

Thanks!

**Comment by dmnfarrell** [comment]
1.Yes I renamed it to pathogenie. If you see any references to the old name let me know.
2. Yes that is just a method that loads the gis sequences into a set of records first. I've updated the notebook at https://github.com/dmnfarrell/teaching/blob/master/sarscov2/protein_annotation.ipynb which I assume you're referring to.
You should run the pip install command again to make sure you have the latest.
If something is not working please tell me.

**Comment by koalap-2018** [comment]
Hi Damien,

Thanks for the comments. Yes very happy to test things out :) Will definitly comment back once i get a chance.
